# Assignment for 28-05-2020

Question 1:Accept a string from the user and count no.of vowels,consonants and special characters

In [4]:
inp=input("Enter a string: ")
vc=0
cc=0
inp.lower()
for i in inp:
    if i in 'aeiou':
        vc+=1
    elif i in " ":
        continue
    else:
        cc+=1
print("vowel count:",vc,"\nconsonants count: ",cc)

Enter a string: bharadwaj
vowel count: 3 
consonants count:  6


Question 2:count the no.of pairs of a in the given string(Infosys InfyTQ)string → "abbaaccbbaaa"output → 2

In [5]:
inp=input("Enter a String: ")
c=0
c1=0
p=0
for i in inp:
    if i=='a' and c==0:
        c=1
    elif i=='a' and c==1:
        c=0
        p=p+1
    else:
        c=0
print(p)

Enter a String: abbaaccbbaaa
2


Question 3:Test case1: string →  "((((()()()))))()" output →  8
           Test case2: string →  "((()))()" Output →  4

In [7]:
inp=input("Enter a String: ")
c=0
c1=0
p=0
for i in inp:
    if i=='(':
        c+=1
    elif i==')' and c>0:
        c-=1
        p=p+1
print(p)

Enter a String: ((((()()()))))()
8


Question 4:Read the input as two limits and Print the output as Palindrome count between the two limits
           Example #1; Input : 1 10 Output : 9
           Example #2: Input : 1 100 Output : 18

In [16]:
n=int(input("enter number"))
s = 0
while True:
    k = str(n)
    if k == k[::-1]:
        print(k)
        break
    else:
      m = int(k[::-1])
      n += m
      s += 1
      print(k)

enter number127
127
848


Question 5:Accept a number as an input and check whether the given number is palindrome or not if it is a palindrome number print the number on the screen if it is not a palindrome number reverse that number and add it to previous number repeat this until will get a palindrome number and print that palindrome number on the screen
input: 127
output: 848

In [14]:
n=int(input("Enter limits: "))
c=0
while c==0:
    t=0
    v=n
    while n>0:
        r=n%10
        t=(t*10)+r
        n=n//10
    if t==v:
        c=1
        break
    n=v+t
print(t)

Enter limits: 127
848


Question 6:input:orange  →   output : puboif

In [15]:
value = 'orange'
var=(chr(ord(value[0])+1))
var1=(chr(ord(value[1])+3))
var2=(chr(ord(value[2])+1))
var3=(chr(ord(value[3])+1))
var4=(chr(ord(value[4])+2))
var5=(chr(ord(value[5])+1))
value=var+var1+var2+var3+var4+var5
print(value)

puboif


Question 7:Accept input as a string and display sum of digits as the output input:Appli123cation456 
Output: 21

In [10]:
n=input("Enter a String: ")
a=[]
for i in n:
    try:
        p=int(i)
        a.append(p)
    except:
        continue
print(sum(a))

Enter a String: Appli123cation456
21


Question 8:Write a program to prompt for a score between 0.0 and 1.0. If the score is out of range, print an error. If the score is between 0.0 and 1.0, print a grade using the following table:
Score Grade
>= 0.9 A
>= 0.8 B
>= 0.7 C
>= 0.6 D
< 0.6 F
If the user enters a value out of range, print a suitable error message and exit. For the test, enter a score of 0.85

In [11]:
score = float(input(" "))
if score>1.0 or score<0.0 :
    print("error")
elif score>=0.9 :
    print ('A')
elif score>=0.8 :
    print( 'B')
elif score>=0.7 :
    print( 'C')
elif score>=0.6 :
    print( 'D')
else :
    print( 'F')
    

 0.85
B


Question 9:Write a Python program which iterates the integers from 1 to 50. For multiples of three print "Fizz" instead of the number and for the multiples of five print "Buzz". For numbers which are multiples of both three and five print "FizzBuzz".
Sample Output :
fizzbuzz
1
2
fizz
4
buzz

In [12]:
for fizzbuzz in range(6):
    if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
        print("fizzbuzz")
        continue
    elif fizzbuzz % 3 == 0:
        print("fizz")
        continue
    elif fizzbuzz % 5 == 0:
        print("buzz")
        continue
    print(fizzbuzz)

fizzbuzz
1
2
fizz
4
buzz


Question 10:Write a Python program to find the median of three values.
Expected Output:
Input first number: 26                                            
Input second number: 15                                                
Input third number: 29                                                  
The median is 26

In [13]:
a = int(input("num1"))
b = int(input("num2"))
c = int(input("num3"))

if b < a and a < c:
    print(a)
elif c < a and a < b:
    print(a)
    
elif c < b and b < a:
    print(b)
elif a < b and b < c:
    print(b)
    
elif b < c and c < a:
    print(c)    
elif a < c and c < b:
    print(c)

num126
num215
num329
26
